### ISSUE TITLE
chore: Make packages/medusa/src/services/claim-item.js pass linting 

### ISSUE THREAD

**Issue by srindom** [medusajs/medusa#499]
In an effort to improve our code quality we want to have make sure that all files in the repo pass linting.

Please see this guide for a step-by-step guide for how to complete this task: https://www.notion.so/medusajs/Lint-challenge-checklist-b4efc761cb36499abf4726cad0dd02e4

🚀

**Comment by ashish-patwal** [comment]
> I was going to take this one but there is a line fixing which seems not to be that straightforward. I'm talking about 153th line [here](https://github.com/medusajs/medusa/blob/chore/lint/packages/medusa/src/services/claim-item.js#L153). `update.metadata` there is passed as a second argument but `update` itself doesn't seem to be defined / imported in the file earlier and linter telling the same thing. I didn't go with any solution for this just to not break the things, probably this case requires some clarification from the ones who knows the codebase.

@chikaldirick i also agree with your point . Recently i linted one of the files which had this same problem . I think the maintainers had been updating the codebase and forget to update the definition there at line [153](https://github.com/medusajs/medusa/blob/chore/lint/packages/medusa/src/services/claim-item.js#L153) . As i can see from the [line 142](https://github.com/medusajs/medusa/blob/chore/lint/packages/medusa/src/services/claim-item.js#L142) the data object has been destructured to capture the value of metadata along with other 3 items . I think there needs to be a correction from `update.metadata` to `metadata` .

**Comment by abhijeet007rocks8** [comment]
@srindom can I work on this issue ??
